dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/FieldResource.java

```diff
@@ -66,6 +62,45 @@ public Response createContentTypeField(@PathParam("typeId") final String typeId,
 		Response response = null;
 		
 		try {
+			final List<Field> fields = new JsonFieldTransformer(fieldsJson).asList();
+
+			for (Field field : fields) {
+				field = fapi.save(field, user);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "field". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)